src/Microsoft.AspNet.Mvc.Core/Formatters/HttpParseResult.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc.Formatters
+{
+    internal enum HttpParseResult
```

Comment:
I'm keeping this internal as it's only used in HttpTokenParsingRules, we can change later if we want to